flareon/src/db.rs

```diff
@@ -263,6 +270,16 @@ pub trait FromDbValue {
     fn from_sqlite(value: SqliteValueRef) -> Result<Self>
     where
         Self: Sized;
+
+    /// Converts the given `Postgresql` database value to a Rust value.
+    ///
+    /// # Errors
+    ///
+    /// This method can return an error if the value is not compatible with the
+    /// Rust type.
+    fn from_postgres(value: PostgresValueRef) -> Result<Self>
+    where
+        Self: Sized;
 }
```

Comment:
Should we consider feature-gating those in the future?

Reply:
Please add issue/note in the project not to forget that

Reply:
Done in https://github.com/orgs/flareon-rs/projects/2?pane=issue&itemId=76687633